wazo_auth/services/saml_config.py

```diff
@@ -0,0 +1,94 @@
+# Copyright 2022-2024 The Wazo Authors  (see the AUTHORS file)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+import logging
+import os
+from xml.etree import ElementTree
+
+from wazo_auth.database.models import Domain, SAMLConfig
+from wazo_auth.plugins.http.saml_config.schemas import saml_config_schema
+from wazo_auth.services.helpers import BaseService
+
+logger = logging.getLogger(__name__)
+
+
+class SAMLConfigService(BaseService):
+    def __init__(self, config, saml_service, dao) -> None:
```

Comment:
Assuming additional imports
```
from wazo_auth.database.queries import DAO
from wazo_auth.services.saml import Config, SAMLService
```

```suggestion
    def __init__(self, config: Config, saml_service: SAMLService, dao: DAO) -> None:
```